<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LNT-HZ-NP-SQA18" ignored="0" name="Production" tests="2" failures="1" timestamp="05 Jun 2021 08:53:15 GMT" time="86.785" errors="0">
  <testcase name="AddNewProduction" time="46.971" classname="testScriptsSmoke.TS_006_Production">
    <skipped/>
  </testcase> <!-- AddNewProduction -->
  <testcase name="AddNewProduction" time="26.47" classname="testScriptsSmoke.TS_006_Production">
    <failure type="java.lang.AssertionError" message="expected [Error !] but found [Success !]">
      <![CDATA[java.lang.AssertionError: expected [Error !] but found [Success !]
at pageFactory.BaseClass.VerifySuccessMessage(BaseClass.java:392)
at pageFactory.TagsAndFoldersPage.CreateFolder(TagsAndFoldersPage.java:202)
at testScriptsSmoke.TS_006_Production.AddNewProduction(TS_006_Production.java:62)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- AddNewProduction -->
</testsuite> <!-- Production -->
